src/open_producten/producttypen/tests/api/test_logging.py

```diff
@@ -0,0 +1,82 @@
+from django.contrib.contenttypes.models import ContentType
+from django.urls import reverse_lazy
+
+from rest_framework import status
+from reversion.models import Version
+
+from open_producten.logging.constants import Events
+from open_producten.logging.models import TimelineLogProxy
+from open_producten.producttypen.models import ProductType
+from open_producten.producttypen.tests.factories import (
+    ThemaFactory,
+    UniformeProductNaamFactory,
+)
+from open_producten.utils.tests.cases import BaseApiTestCase
+
+
+class TestProductTypeLogging(BaseApiTestCase):
+
+    def setUp(self):
+        super().setUp()
```

Comment:
```suggestion
```

Reply:
removed the whole file, the tests were already added to the product & producttype api tests